docs/security/trust/content_trust.md

```diff
@@ -152,23 +152,38 @@ To enable content trust in a `bash` shell enter the following command:
 export DOCKER_CONTENT_TRUST=1
 ```
 
-Once set, each of the "tag" operations require key for trusted tag. All of these
-commands also support the `--disable-content-trust` flag. This flag allows
-publishers to run individual operations on tagged images without content trust on an
-as-needed basis. 
+Once set, each of the "tag" operations requires a key for a trusted tag.
+
+In an environment where `DOCKER_CONTENT_TRUST` is set, you can use the
+`--disable-content-trust` flag to run individual operations on tagged images
+without content trust on an as-needed basis.
+
+```bash
+$  docker pull --disable-content-trust docker/trusttest:untrusted
+```
+
+You can also use this flag in a shell where you haven't enabled trust to turn a
```

Comment:
I'm with @endophage here. The docs simply need to state that --disable-content-trust=false will invoke a command with content trust enabled (regardless of how the ENV variable is set).